device/CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ set(UMD_DEVICE_SRCS
     wormhole/wormhole_coordinate_manager.cpp
     pcie/pci_device.cpp
     hugepage.cpp
+    ../common/logger_.cpp # ../common looks a bit strange
```

Comment:
Do we intend to remove the other logger.hpp ?
And to rewrite all logs in tt_silicon_driver?

Reply:
Yes/yes, just not in this PR.
Wanted to get the basic infrastructure reviewed by the team before imposing it everywhere.